JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -440,10 +448,7 @@ - (UICollectionViewCell *)collectionView:(JSQMessagesCollectionView *)collection
     id<JSQMessageData> messageItem = [collectionView.dataSource collectionView:collectionView messageDataForItemAtIndexPath:indexPath];
     NSParameterAssert(messageItem != nil);
 
-    NSString *messageSenderId = [messageItem senderId];
-    NSParameterAssert(messageSenderId != nil);
```

Comment:
can, but should we. Isn't that asserting an implementation detail of `isOutgoingMessage`?